src/core/crypto/Utilities.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
review name of this file `Utilities` doesn't make sense for a `getHasher` exported function.